README.md

```diff
@@ -8,6 +8,11 @@ This repository should be used in conjunction with these following repos :
 - [Mobile Client Demo App](https://github.com/feedhenry-raincatcher/raincatcher-demo-mobile)
 - [WFM Cloud App](https://github.com/feedhenry-raincatcher/raincatcher-demo-cloud)
 
+
+## Setup 
+  This service requires `FH_MONGODB_CONN_URL` environment variable to be a valid mongodb connection string.
+  It can be made available through upgrading database in the Data Browser part of RHMAP.
```

Comment:
Might be good to add a link to the data browser upgrade docs

https://access.redhat.com/documentation/en/red-hat-mobile-application-platform-hosted/3/single/server-side-developer-guide/#upgrading-the-database